tests/Unit/Http/Form/SignupFormTest.php

```diff
@@ -286,7 +286,58 @@ public function lastNameProvider(): array
     }
 
     /**
-     * Test that the Joind.in URL is being validated correctly
+     * Test that the joind.in username is being validated correctly
```

Comment:
Not sure, perhaps we can even remove the short description?

Reply:
Agreed